samples/getting_started/Measurement.qs

```diff
@@ -1,28 +1,33 @@
-/// # Sample
-/// Measurement
+/// # Summary
+/// Measurement sample
 ///
 /// # Description
+/// This Q# program demonstrates how to perform measurements in Z basis.
+///
+/// # Remarks
 /// Quantum measurement is an irreversible operation in which a quantum system
 /// is manipulated to yield a numerical result. Measuring a quantum system
 /// generally changes the quantum state that describes that system.
-///
-/// In Q#, the result of a measurement is a value of the type `Result`, that is,
-/// `One` or `Zero`.
-///
-/// This Q# program exemplifies different types of measurements.
-import Std.Measurement.*;
-
 operation Main() : (Result, Result[]) {
+    use q = Qubit(); // Allocate a qubit. Qubit is in |0〉 state after allocation.
```

Comment:
Here too (and in the other files) I think we should stick to the pattern of having comments above the lines they refer to.